mie2c/e2c.py

```diff
@@ -135,6 +140,7 @@ def forward(self, x):
             # x = self.batch_norm_layers[ii](x)
             if self.conv_activation:
                 x = self.conv_activation(x)
+        x = self.sigmoid(x)
```

Comment:
Should the _sigmoid_ activation only be applied for the loss function so that the forward pass can still yield an RGB image?

Reply:
you can still have RGB but normalized to 1. RGB is more about having 3 channel than about what scale they are on.